src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Day/DiaryDayViewModel.swift

```diff
@@ -0,0 +1,71 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import Combine
+
+class DiaryDayViewModel {
+
+	// MARK: - Init
+
+	init(day: DiaryDay, store: DiaryStoring) {
+		self.day = day
+		self.store = store
+
+		store.diaryDaysPublisher
+			.sink { [weak self] days in
+				guard let day = days.first(where: { $0.dateString == day.dateString }) else { return }
+
+				self?.day = day
```

Comment:
can the day switch while I have opened a specific day?

Reply:
Not the day but it's entries can change